src/components/LocationAutocompleteInput/LocationAutocompleteInputImpl.js

```diff
@@ -1,11 +1,18 @@
 import React, { Component } from 'react';
-import { any, arrayOf, bool, func, number, shape, string, oneOfType } from 'prop-types';
+import { any, arrayOf, bool, func, number, shape, string, oneOfType, object } from 'prop-types';
 import classNames from 'classnames';
 import debounce from 'lodash/debounce';
 import { propTypes } from '../../util/types';
-import { getPlacePredictions, getPlaceDetails } from '../../util/googleMaps';
+import Geocoder from './GeocoderGoogleMaps';
 
-import css from './LocationAutocompleteInputGoogleMaps.css';
+import css from './LocationAutocompleteInput.css';
+
+// When using the Google Maps Places API for geocoding, a "Powered by
+// Google" logo should be shown next to the results. Turn this to
+// `false` when using some other Geocoding API. Autocomplete
+// predictions dropdown bottom padding might need adjusting as well to
+// hide the space left for the logo.
+const SHOW_POWERED_BY_GOOGLE = true;
```

Comment:
Would it be safer to use `config.googleMapsAPIKey` check somehow here? Especially if we end up making mapbox our default map provider? (Missing to turn this on/off is a clear violation of their terms.)

Reply:
Even better would be that the whole div (or just the flag) is exported from Geocoder file.

Reply:
Could be better but a config var might make customisers think that it is enough to change the provider.

I think a config still makes sense, then we can already adjust the predictions list paddings etc. based on that. We just have to mention that some imports have to be changes too.

Reply:
I moved the attribution element to the geocoder.

However, the predictions list leaves bottom padding for the attribution, and that padding is added in many places. I added a global var that can be used to set the padding when the map provider is changed.